pkg/api/handlers/compat/containers_logs.go

```diff
@@ -148,7 +148,7 @@ func LogsFromContainer(w http.ResponseWriter, r *http.Request) {
 			frame.WriteString(line.Time.Format(time.RFC3339))
 			frame.WriteString(" ")
 		}
-		frame.WriteString(line.Msg)
+		frame.WriteString(line.Msg + "\n")
```

Comment:
I think we should only do it for the compat layer:
```golang
if !utils.IsLibpodRequest(r) {
....
```

@ashley-cui WDYT?

Reply:
That sounds good to me

Reply:
@vrothberg @ashley-cui do you rather want:

```go
		if utils.IsLibpodRequest(r) {
			frame.WriteString(line.Msg)
		} else {
			frame.WriteString(line.Msg + "\n")
		}
``` 

or 

```
		frame.WriteString(line.Msg)
		if !utils.IsLibpodRequest(r) {
			frame.WriteString("\n")
		}
```

Reply:
The latter seems cleaner to me. Also please add a code comment (ideally with a link to the issue you're fixing) as well.

Reply:
Added and squashed @vrothberg